packs/linux/actions/iostat.yaml

```diff
@@ -0,0 +1,24 @@
+name: "iostat"
+runner_type: "run-remote"
+description: "returns I/O stats of system storage"
+enabled: true
+entry_point: ""
+parameters:
+  count:
```

Comment:
Looks like this parameter is not used in args.

Besides that I tested the action and it looks good to me.